src/routes.js

```diff
@@ -21,6 +23,7 @@ export default (
         <Switch>
                <Route exact path="/" component={Donors} />
                 <Route path="/donor/create" component={CreateDonor} />
+                <Route path={"/donor/:donorId/edit"} render={(props)=> <DonationList {...props} />}  component={EditDonor} />
```

Comment:
I don't think you need to pass `render={(props)=> <DonationList {...props} />}`

Reply:
Ok, I can remove this....wasn't sure it what this actually was, but saw you had it below and that I needed the donor info passed